ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -298,8 +363,30 @@ open class SegmentedControl: UIControl {
         var rightOffset: CGFloat = 0
         var leftOffset: CGFloat = 0
         for (index, button) in buttons.enumerated() {
-            if style.segmentsHaveEqualWidth {
-                rightOffset = UIScreen.main.roundToDevicePixels(CGFloat(index + 1) / CGFloat(buttons.count) * frame.width)
+            if segmentsHaveEqualWidth {
+                if (style == .tabs)
+                {
+                    rightOffset = UIScreen.main.roundToDevicePixels(CGFloat(index + 1) / CGFloat(buttons.count) * frame.width)
```

Comment:
I think we should try to get the [UIWindowScene screen](https://developer.apple.com/documentation/uikit/uiwindowscene/3198089-screen) and default to the UIScreen.main so we can support multi window scenarios:

`let screen = window?.windowScene?.screen ?? UIScreen.main`

I also noticed that in other places in this file, could you fix them too?